src/tokenize.c

```diff
@@ -82,12 +82,12 @@ uint32_t simpleTokenizer_Next(RSTokenizer *base, Token *t) {
     char *tok = toksep(&self->pos, &origLen);
     // normalize the token
     size_t normLen = origLen;
-    if (normLen > MAX_NORMALIZE_SIZE) {
-      normLen = MAX_NORMALIZE_SIZE;
-    }
     char normalized_s[MAX_NORMALIZE_SIZE];
     char *normBuf;
     if (ctx->options & TOKENIZE_NOMODIFY) {
+      if (normLen > MAX_NORMALIZE_SIZE) {
```

Comment:
Is this unreachable code?
It seems that we are creating the tokenizer with `TOKENIZE_NOMODIFY` but we are clearing it on `Start`

https://github.com/RediSearch/RediSearch/blob/987128879f49f766718f30665300d4f4bb7a7cd2/src/fragmenter.c#L121-L122

Reply:
Yes this is unreachable, TOKENIZE_NOMODIFY is not set.
Either way,  `FragmentList_FragmentizeBuffer` is only called from testing, so this part is a dead code.
I'm adding a comment to clarify.